feature_tests/features/producer/producer-supersedeDocumentPointer-failure.feature

```diff
@@ -1,4 +1,4 @@
-Feature: Producer Supersede Failure scenarios
+Feature: Producer Supersede Edge Case scenarios
```

Comment:
Don't we still need all of the failure tests? Also not sure that the new error message is being tested here is it?

Reply:
No idea how this ended up here as 'Edge Cases'